app/models/user.rb

```diff
@@ -1,11 +1,24 @@
 class User < ApplicationRecord
   devise :omniauthable, :timeoutable
+
   include Reauthable
 
+  before_create do
+    self.invitation_expires_at = Rails.configuration.x.auth.reauthenticate_in.from_now
+  end
+
+  scope :pending_activation, lambda {
```

Comment:
Wondering if it would be good to add a spec for the scopes here.

Reply:
done